ZuluContent/Spells/First/Feeblemind.cs

```diff
@@ -12,20 +15,27 @@ public FeeblemindSpell(Mobile caster, Item spellItem) : base(caster, spellItem)
 
         public async Task OnTargetAsync(ITargetResponse<Mobile> response)
         {
-            if (!response.HasValue)
+            if (!response.HasValue || !(response.Target is IBuffable buffable))
```

Comment:
Nitpick: Is it needed to use pattern matching when just returning?

Reply:
It's a bit unintuitive but the pattern matching here is declaring the `buffable` property for the rest of the function.

the alternative would be like 
```csharp
            if (!response.HasValue || !(response.Target is IBuffable))
                return;

            var buffable = (IBuffable)response.Target;
```

Reply:
oh yeah right I see